lib/Fmt_ast.ml

```diff
@@ -3116,7 +3120,8 @@ and fmt_class_params c ctx params =
     (hvbox 0
        (wrap_fits_breaks c.conf "[" "]" (list_fl params fmt_param) $ fmt "@ ") )
 
-and fmt_type_declaration c ?ext ?(pre = "") ctx ?fmt_name ?(eq = "=") decl =
+and fmt_type_declaration c ?ext ?(pre = "") ctx ?fmt_name ?(eq = "=")
+    ~force_before decl =
```

Comment:
It might be worth to document this new argument and to reference `force_docstring_before`.